src/transport/FabricTable.cpp

```diff
@@ -174,9 +174,13 @@ CHIP_ERROR FabricInfo::FetchFromKVS(PersistentStorageDelegate * kvs)
 CHIP_ERROR FabricInfo::GetCompressedId(FabricId fabricId, NodeId nodeId, PeerId * compressedPeerId) const
 {
     ReturnErrorCodeIf(compressedPeerId == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
-    CompressedFabricId compressedFabricId = 0;
-    MutableByteSpan fabricIdSpan(reinterpret_cast<uint8_t *>(&compressedFabricId), sizeof(compressedFabricId));
-    ReturnErrorOnFailure(GenerateCompressedFabricId(mRootPubkey, fabricId, fabricIdSpan));
+    uint8_t compressedFabricIdBuf[sizeof(CompressedFabricId)];
```

Comment:
should we use sizeof(uint64_t) here to make it explicit?

Using CompressedFabricId makes it look a lot like a strong and below I see that it is just a Get64.